modules/management/src/main/java/org/mule/module/management/agent/DefaultJmxSupportAgent.java

```diff
@@ -116,13 +117,13 @@ public void initialise() throws InitialisationException
             // there must be only one jmx agent, so lookup by type instead
             if (registry.lookupObject(AbstractJmxAgent.class) == null)
             {
-                registry.registerAgent(agent);
```

Comment:
no  , spring now applies lifefyclle correclty and everything is correctly registered